managed/cmd/pmm-managed/main.go

```diff
@@ -558,15 +559,15 @@ func setup(ctx context.Context, deps *setupDeps) bool {
 	deps.l.Infof("Updating supervisord configuration...")
 	settings, err := models.GetSettings(db.Querier)
 	if err != nil {
-		deps.l.Warnf("Failed to get settings: %+v.", err)
+		deps.l.Warnf("Failed to get settings: %s.", err)
```

Comment:
It was a bit weird to see a part of the stack trace printed at the start of pmm-managed.

`
[2023-06-26T14:35:15.947+00:00] Failed to get Percona SSO Details: PMM Server is not connected to Portal
github.com/percona/pmm/managed/models.init
        /home/builder/rpm/BUILD/pmm-7a7431afdc35098f665dbd8d33ccf914fad8b0b5/src/github.com/percona/pmm/managed/models/percona_sso_model_helpers.go:35
runtime.doInit
        /usr/local/go/src/runtime/proc.go:6507
runtime.doInit
        /usr/local/go/src/runtime/proc.go:6484
runtime.main
        /usr/local/go/src/runtime/proc.go:233
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1598.  ESC[33mcomponentESC[0m=setup
`